irr/src/OpenGL/ExtensionHandler.h

```diff
@@ -168,7 +168,14 @@ class COpenGL3ExtensionHandler : public COGLESCoreExtensionHandler
 			// Since our texture strings can get quite long we also truncate
 			// to a hardcoded limit of 82
 			len = std::min(len, std::min(MaxLabelLength, 82U));
+			// According to KHR_debug specification
+			// https://registry.khronos.org/OpenGL/extensions/KHR/KHR_debug.txt
+			// KHR suffix is used for entry point in OpenGL ES.
+#ifndef _IRR_COMPILE_WITH_OGLES2_
 			GL.ObjectLabel(identifier, name, len, label);
+#else
+			GL.ObjectLabelKHR(identifier, name, len, label);
```

Comment:
considering the GL driver does the following
```c
	KHRDebugSupported = isVersionAtLeast(4, 6) || queryExtension("GL_KHR_debug");
```
always using the KHR suffix is not necessarily correct